mewarpx/mewarpx/utils_store/init_restart_util.py

```diff
@@ -17,20 +17,23 @@ def clean_old_checkpoints(prefix, directory):
                 print(f"Removing old checkpoint file {d}")
                 shutil.rmtree(os.path.join(directory, d))
 
-def run_restart(checkpoint_directory="diags", checkpoint_prefix=default_checkpoint_name,
+def run_restart(checkpoint_directory="diags",
+                checkpoint_prefix=default_checkpoint_name,
                 force=False, additional_steps=None):
     '''
     Attempts to restart a run by looking for checkpoint files
     starting with a prefix in the given directory.
 
     Arguments:
-        checkpoint_directory (string): Look in this directory for checkpoint directories. Default is 'diags'.
-        checkpoint_prefix (string): Look for a checkpoint directory starting with this prefix
-          to restart from.
-        force (bool): If true, a problem with restarting from a checkpoint will cause an error,
-          otherwise simply print a warning.
-        additional_steps (int): The number of steps to run after restarting from the checkpoint.
-          If this is None then it will run to the current value of mwxrun.simulation.max_steps.
+        checkpoint_directory (string): Look in this directory for checkpoint
+            directories. Default is 'diags'.
```

Comment:
```suggestion
            directories. Default is ``diags``.
```
Sphinx code highlighting syntax.